samples/bluetooth/hci_usb/src/main.c

```diff
@@ -416,8 +416,13 @@ static void btusb_int_in(u8_t ep, enum usb_dc_ep_cb_status_code ep_status)
 static void btusb_bulk_out(u8_t ep, enum usb_dc_ep_cb_status_code ep_status)
 {
 	struct net_buf *buf;
-	u32_t len, read = 0;
+	u32_t len;
+#ifdef CONFIG_SOC_FAMILY_QUARK
+	u32_t read = 0;
```

Comment:
That is already handled in the usb_dc_ep_read_wait() in usb_dc_dw.c driver. So you can remove CONFIG_SOC_FAMILY_QUARK part.

Reply:
Okay, I'll remove all the Quark specific stuff since you say it's already handled in DW driver.